index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
+</html>
```

Comment:
- [ ] OPTIONAL: Your code looks good and you implemented it as required, but It would be better if you made some indentations to improve the code's readability.
```suggestion
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="stylesheet" href="style.css">
        <title>Document</title>
    </head>
    <body>
        <h1>Hello Microverse!</h1>
    </body>
</html>
```

Reply:
thanks for the review, will make the indentation changes @Kandy-Peter